consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/ConsistencyChecker.java

```diff
@@ -202,7 +202,9 @@ public void run() {
             findRequest.where(and(
                     withValue("configurationName = " + configuration.getConfigurationName()),
                     withValue("whenAvailableDate <= " + ClientConstants.getDateFormat().format(new Date())),
-                    not(withSubfield("jobExecutions", withValue("completedFlag = true")))));
+                    not(withSubfield("jobExecutions", withValue("jobStatus = 'COMPLETED_SUCCESS'"))),
+                    not(withSubfield("jobExecutions", withValue("jobStatus = 'COMPLETED_PARTIAL'")))
```

Comment:
Only one execution should ever be processed at a time.  But that's something that has to be handled somewhere else because of how updates to the migrationJob entity are done.  This query is just to get a collection of what is processable based on what is in the db at the time the query was executed.